SPECS/kata-containers/runtime-Re-generate-the-client-code.patch

```diff
@@ -1,1401 +0,0 @@
-From 2edfd719d46f26c85882944c64ebed3f315d72df Mon Sep 17 00:00:00 2001
```

Comment:
[This](https://github.com/kata-containers/kata-containers/commit/2edfd719d46f26c85882944c64ebed3f315d72df) is cherry picked from https://github.com/kata-containers/kata-containers/commit/5d63fcf344808495d3a67f1f718a7f0956bac87f, which is already merged in 3.1.0
<img width="531" alt="image" src="https://user-images.githubusercontent.com/30801614/234128141-dbc8f0e2-fd64-415a-b72d-170a74326b0a.png">